MinPlatformPkg/Test/Library/TestPointCheckLib/StandaloneMmTestPointCheckLib.inf

```diff
@@ -0,0 +1,63 @@
+## @file
+# Component information file for the Standalone MM Test Point Check Library
+#
+# Copyright (c) 2017-2019, Intel Corporation. All rights reserved.<BR>
+# Copyright (c) Microsoft Corporation.
+#
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = StandaloneMmTestPointCheckLib
+  FILE_GUID                      = AE975D85-61EA-48F1-A68B-BA777751AC21
+  MODULE_TYPE                    = MM_STANDALONE
+  VERSION_STRING                 = 1.0
+  PI_SPECIFICATION_VERSION       = 0x00010032
+  LIBRARY_CLASS                  = TestPointCheckLib|MM_STANDALONE
+  CONSTRUCTOR                    = StandaloneMmTestPointCheckLibConstructor
+
+[LibraryClasses]
```

Comment:
It looks like you may want to include `PcdLib` (with the PCDs define below). It might be pulled in when this library is linked so it's not causing a problem now, but the library should account for its own dependencies.